playbooks/ldap-validator/viyaldapvalidator.yml

```diff
@@ -43,8 +43,8 @@
   - name: Assert that the most important variables are defined
     assert:
       that:
-        - "{{ item }} is defined"
-        - "{{ item }} | trim != '' "
+        - "item is defined"
+        - "item | trim != '' "
```

Comment:
since these are quoted they will likely evaluate as true because they are string with non-zero length, not as variable expanded values and conditionals

Reply:
I see what you are saying.   Let me try a change.

Reply:
Check latest changes.  quotes removed.